src/logger.cpp

```diff
@@ -42,7 +34,7 @@ void Logger::installHandler()
     originalHandler = qInstallMessageHandler(handleMessage); // This function returns the previous message handler
 
     if (qEnvironmentVariableIsEmpty("QT_MESSAGE_PATTERN")) {
-        qSetMessagePattern(QStringLiteral("%{time [hh:mm:ss.zzz]} %{message}"));
```

Comment:
Was adding this hunk intentional? What does this do?

Reply:
Ops, my bad, I was using a personal pattern while debugging.